docs/source/setup.rst

```diff
@@ -38,3 +41,25 @@ If you plan to use ROS2, you can similarly create a bash function like this:
     pyrobosim_ros() {
        source /path/to/pyrobosim/setup/setup_pyrobosim.bash humble
     }
+
+
+Docker Setup
+------------
+
+We also provide Docker images compatible with ROS2 Foxy and Humble releases.
+
+If you already have sourced ROS2 in your system (e.g., `source /opt/ros/humble/setup.bash`),
+then you should have a `ROS_DISTRO` environment variable set. Otherwise,
+
+::
+
+    export ROS_DISTRO=humble
+    ./docker/build_docker.bash
+    ./docker/run_docker.bash
+
+Alternatively, you can directly a command from Docker:
+
+::
+
+    ./docker/run_docker.bash "ros2 run pyrobosim_ros demo.py"
```

Comment:
Add 

In another terminal
`docker/run_docker.bash "ros2 run pyrobosim_ros demo_commands.py"`
To see the simulator in action.